### PR TITLE
Added Google Photos Presence!

### PR DESCRIPTION
**Pull request by JohanRBX**
unknown
unknown (2)

### CODE REVIEW

**Comment by SlowLife1661** [comment]
Use 3rd person instead of 1st person

File: presence.ts [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      const presenceData: PresenceData = {
+        details: "Explore",
+        state: "Searching...",
+        largeImageKey: "photos"
+      };
+    presence.setActivity(presenceData);
+    
+
+
+    // Página de compartidos
```
Comment (SlowLife1661) [inline presence.ts:64]: Remove spaces

File: presence.ts [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      const presenceData: PresenceData = {
+        details: "Photos",
+        state: "Looking at a photo...",
+        largeImageKey: "photos",
+        startTimestamp: time
+      };
+      presence.setActivity(presenceData);
+
+
+    // Explorar.
```
Comment (SlowLife1661) [inline presence.ts:29]: Same here

File: presence.ts [hunk 3]
```diff
@@ -0,0 +1,279 @@
+var presence = new Presence({
+    clientId: "820332677177671682"
+  });
+  
+  const time = Math.floor(Date.now() / 1000);
+  const buttons = presence.getSetting("buttons")
+  presence.on("UpdateData", async () => {
+
+  
+    // Fotos
```
Comment (SlowLife1661) [inline presence.ts:10]: Here

File: presence.ts [hunk 4]
```diff
@@ -0,0 +1,279 @@
+var presence = new Presence({
+    clientId: "820332677177671682"
+  });
+  
+  const time = Math.floor(Date.now() / 1000);
+  const buttons = presence.getSetting("buttons")
```
Comment (SlowLife1661) [inline presence.ts:6]: No var, marge const

File: presence.ts [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+
+  
+    // Fotos
+    if (document.location.pathname == "/") {
+      const homepagePresence: PresenceData = {
+        details: "Photos",
+        state: "Viewing your photos",
+        largeImageKey: "photos",
+        startTimestamp: time
+      };
```
Comment (SlowLife1661) [inline presence.ts:17]: Declare this outside scope and you can just reassign it's value

File: presence.ts (hunk 1 above)
Comment (SlowLife1661) [inline presence.ts:64]: Spaces

File: presence.ts [hunk 6]
```diff
@@ -0,0 +1,279 @@
...
+    };
+    presence.setActivity(presenceData);
+  } else if (document.location.pathname.startsWith("/activities/photos_likes")) {
+    const presenceData: PresenceData = {
+      details: "Activity Register",
+      state: "Checking your likes...",
+      largeImageKey: "photos",
+      startTimestamp: time
+    };
+    presence.setActivity(presenceData);
```
Comment (SlowLife1661) [inline presence.ts:276]: Use it once at the end

File: presence.ts [hunk 7]
```diff
@@ -0,0 +1,279 @@
...
+      const presenceData: PresenceData = {
+        details: "Paper bin",
+        state: "Checking the trash...",
+        largeImageKey: "photos",
+        startTimestamp: time
+      };
+      presence.setActivity(presenceData);
+
+
+    // Configuración
```
Comment (SlowLife1661) [inline presence.ts:234]: Spaces

File: presence.ts [hunk 8]
```diff
@@ -0,0 +1,279 @@
...
+      const presenceData: PresenceData = {
+        details: "Archive",
+        state: "Reviewing a file ...",
+        largeImageKey: "photos",
+        startTimestamp: time
+      };
+      presence.setActivity(presenceData);
+
+
+    // Papelera
```
Comment (SlowLife1661) [inline presence.ts:223]: Spaces

File: presence.ts [hunk 9]
```diff
@@ -0,0 +1,279 @@
...
+      const presenceData: PresenceData = {
+        details: "Utilities",
+        state: "Creating a movie ...",
+        largeImageKey: "photos",
+        startTimestamp: time
+      };
+      presence.setActivity(presenceData);
+
+
+    // Archivo.
```
Comment (SlowLife1661) [inline presence.ts:204]: Spaces

File: presence.ts [hunk 10]
```diff
@@ -0,0 +1,279 @@
...
+      const presenceData: PresenceData = {
+        details: "Albums",
+        state: "Looking at an album...",
+        largeImageKey: "photos",
+        startTimestamp: time
+      };
+      presence.setActivity(presenceData);
+
+
+    // Página de utilidades.
```
Comment (SlowLife1661) [inline presence.ts:185]: Spaces

File: presence.ts [hunk 11]
```diff
@@ -0,0 +1,279 @@
...
+                label: "View",
+                url: "https://photos.google.com/canvas"
+            }
+        ]
+    };
+    presence.setActivity(presenceData);
+
+
+
+    // Álbumes
```
Comment (SlowLife1661) [inline presence.ts:166]: Spaces

**Review by SlowLife1661 — CHANGES_REQUESTED** [review]
*Just a suggestion* And you can just remove the comments and the code

File: presence.ts (hunk 4 above)
Comment (SlowLife1661) [reply presence.ts:6]: And the settings what happens if they change it?

**Review by Bas950 — CHANGES_REQUESTED** [review]
first of all put it in the correct folder

**Pull request closed without merging**